source/documentation/query/lateral-join.md

```diff
@@ -16,22 +16,26 @@ sub-patterns.
 Another way to think of a lateral join is as a `flatmap`.
 
 Examples:
-```
+
+```sparql
 ## Get exactly one label for each subject with type `:T`
 SELECT * {
-   ?s rdf:type :T
-   LATERAL {
-     SELECT * { ?s rdfs:label ?label } LIMIT 1
-   }
+  ?s rdf:type :T
+  LATERAL {
+    SELECT * { ?s rdfs:label ?label } LIMIT 1
+  }
 }
 ```
 
-```
+```sparql
 ## Get zero or one labels for each subject.
 SELECT * {
-   ?s ?p ?o
-   LATERAL { OPTIONAL { SELECT * ?s rdfs:label ?label } LIMIT 1}
```

Comment:
Hmm, I think the `{}'s` were not balanced here?

Reply:
Thanks for the fix.